nodeJS/express/forms_and_data_handling.md

```diff
@@ -402,32 +403,32 @@ And we need to update our `users.ejs` view to render these errors. Let's create
 <% } %>
 ```
 
-And we'll include the partial just above our form in `users.ejs`.
+And we'll include the partial just above our form in `create_user.ejs`.
 
 ```ejs
 <%- include("partials/errors.ejs") %>
 ```
 
 If the form is filled out incorrectly, you'll see the page contains all the errors.
 
-Now that we can create users, we also need a way to modify them. Let's create a form that lets us specify an ID and the values we want to update by making a new view: `update.ejs`.
+Now that we can create users, we also need a way to modify them. Let's create a form that lets us specify an ID and the values we want to update by making a new view: `update_user.ejs`.
 
 ```ejs
-<!-- views/update.ejs -->
+<!-- views/update_user.ejs -->
 <%- include("partials/errors.ejs") %>
-<form action="/users/<%= user.id %>/update" method="POST">
+<form action="/<%= user.id %>/update" method="POST">
   <input type="text" name="firstName" value="<%= user.firstName %>" required>
   <input type="text" name="lastName" value="<%= user.lastName %>" required>
   <button type="submit">Update User</button>
 </form>
 ```
 
-We'll modify `index.ejs` to include a new "Update" button to make our `GET` request to the route:
+We'll modify `index.ejs` to include a new "Update" button next to each user to make our `GET` request to the route:
 
 ```ejs
 <li>
   ID: <%= user.id %>, Name: <%= user.firstName %> <%= user.lastName %>
-  <a href="/users/<%= user.id %>/update">Update</a>
```

Comment:
With the changes to the route paths, a leading `/users` wouldn't be needed.
However, I can certainly make those update paths manually have a leading `/users` if you feel that makes more sense.

The original justification was that since the entire mini-app is just about users, then `/` is implicitly all about users, hence other paths just being `/create` instead of `/users/create` etc.

Reply:
I can see the logic for the justification here, so I'm good with this being changed. The original intent was to distinguish `users` as a group and imply that other "things" could be created the same way. But given this example is strictly related to users, I can also see the reasoning for wanting to remove the redundancy. I'm okay with this provided the code has been tested (won't be able to test til later today myself so I'll trust your judgement lol)

Reply:
I've tested this on my end and copied the code over to the examples, though there's of course every chance I've still made a booboo. If you do get the chance to test run the code, that'd be much appreciated!